eggplant/profiles/models.py

```diff
@@ -4,7 +4,16 @@
 from django.dispatch.dispatcher import receiver
 from django.db.models.signals import post_save
 from django.contrib.auth.models import User
+from django.core.files.storage import FileSystemStorage
+import os
 
+# Prepare profile photo dir
+profile_photo_dir = settings.BASE_DIR + "/media/profile/"
```

Comment:
In core.utils there is a function called `generate_upload_path` it may make sense to use it to have random file names.